src/biogeochem/CropType.F90

```diff
@@ -570,13 +589,13 @@ subroutine CropUpdateAccVars(this, bounds, t_ref2m_patch, t_soisno_col)
        call endrun(msg=errMsg(sourcefile, __LINE__))
     endif
 
-    ! Accumulate and extract GDDPLANT
+    ! Accumulate and extract HUI and GDDACCUM
     
-    call extract_accum_field ('GDDPLANT', rbufslp, nstep)
+    call extract_accum_field ('HUI', rbufslp, nstep)
```

Comment:
Does GDDACCUM also need to be added here? It seems that HUI and GDDACCUM are both added in most other places throughout the code, so I want to make sure it's not an oversight to leave it out here.

Reply:
I'll answer this one, since this is related to something I asked @samsrabin to change: GDDACCUM does not need to be updated here. However, the reason for this is subtle, and the fact that its absence caught your eye (thank you for your careful look that led to that!) suggests that there should probably be a comment here explaining why this is needed for HUI. Something like: "HUI needs this special update (which is non-standard for accumulation fields) because it can be changed outside of the accumulation routines (in CropPhenology), so the accumulation buffer needs to be reset when that happens." @samsrabin what do you think?

Reply:
Good idea, done (66abdbc).